js/services/JobDetailsService.js

```diff
@@ -58,6 +58,10 @@ define(['appConfig', 'services/job/jobDetail', 'atlas-state', 'services/http'],
 					case 'PLP':
 						return 'plp/' + n.jobParameters.cohortId;
 				}
+			case 'generateEstimationAnalysis':
+				return 'estimation/cca/' + n.jobParameters.estimation_analysis_id + '/executions';
```

Comment:
Please forward the user to concrete executions. Not just execution page

Reply:
Since we don't have a separate generation page, I'd suggest to redirect a user to generations tab and:
- expand the section (table) of Source used by the generation
- scroll page to the generation line
- highlight (change background) the line with bootstrap's `warning` class

(when a user clicks the notification)